src/test/java/dk/kb/datahandler/kaltura/KalturaDeltaUploadIntegrationTest.java

```diff
@@ -0,0 +1,74 @@
+package dk.kb.datahandler.kaltura;
+
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+import org.apache.solr.common.SolrDocumentList;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.kaltura.client.enums.MediaType;
+
+import dk.kb.datahandler.config.ServiceConfig;
+
+@Tag("integration")
+public class KalturaDeltaUploadIntegrationTest {
+
+    
+    private static final Logger log = LoggerFactory.getLogger(KalturaDeltaUploadIntegrationTest.class);
+    
+    @BeforeAll
+    static void setup() {
+        try {
+       ServiceConfig.initialize("conf/ds-datahandler-behaviour.yaml","ds-datahandler-integration-test.yaml");
+        }
+        catch(Exception e) {
+            log.error("error loading behaviour+integration test yaml");
+        }
+    }
+    
+    @Test
+    void fetchSolrDocumentTest() throws Exception {
+        try {
+        SolrDocumentList records = KalturaDeltaUploadJob.fetchSolrRecords(0, 10);
+        assertTrue(records.size() >= 0);
+        }
+        catch(Exception e) {
+            fail("Solr call failed");
+        }        
+    }
+    
+
+    @Test
+    void uploadStreamTest() throws Exception {
+        try {
+
+       //It is inteded this is a local file. After upload it must be delete in the KMC            
+       String filePath="/home/teg/Music/Boz_-_BBB_20191202.mp3";            
```

Comment:
Could we provide this integration test file in aegis, so that everyone can run it? Can it be deleted programatically when the test has ended?

Reply:
The point of unit and integration tests are that they are run when asked for. If running the test requires manual activity, then at some point the test WILL be forgotten as it we forget that it is here. I dont like that we cannot have it done programatically. 

Could we use a smaller test file and implement automatic deletion of the test file after it's been run?

Reply:
added small mp3 to aegis and integration will test kaltura entry as part of integration-test